app/linux/BUILD.gn

```diff
@@ -47,11 +52,11 @@ group("dist_resources") {
 extract_symbols("brave_symbols") {
   binary = "$brave_dist_dir/brave"
 
-  if (current_cpu == "x86") {
+  if (target_cpu == "x86") {
     # Use "ia32" instead of "x86" for GYP compat.
     symbol_file = "$root_out_dir/brave.breakpad.ia32"
   } else {
-    symbol_file = "$root_out_dir/brave.breakpad.$current_cpu"
+    symbol_file = "$root_out_dir/brave.breakpad.$target_cpu"
   }
 
   deps = [ "//brave:brave_dist_resources" ]
```

Comment:
here you're referring to `current_cpu` target implicitly. This may quickly get out of control.

what issue exactly are you solving? Do you have an example where things go wrong?

Reply:
So interestingly I found two conflicting gn docs now that say opposite things about using current vs target, but these are specifically outputs that we only care about for the target so I think it still makes sense here

Reply:
> I'm not following. target_cpu is what you want in almost every case outside of the toolchain itself. What exactly would get out of control? We only want outputs for the target_cpu.

> In many cases, this is the same as "host_cpu", but in the case of
  cross-compiles, this can be set to something different. This value is
  different from "current_cpu" in that it does not change based on the current
  toolchain. When writing rules, "current_cpu" should be used rather than
  "target_cpu" most of the time.

`current_cpu` is derived from `current_toolchain`. `current_toolchain` is implied in any `deps` where you don't specify the toolchain explicitly.

`target_cpu` is fixed and is equal to what you passed to `gn`.

Reply:
Actually I just switched things in my head apparently and we do want current most of the time so need to check for possible incorrect target usages instead

Reply:
so now that I've refreshed my memory, I think this change still makes sense because we would never want to build symbols for the host toolchain, but it requires a few other changes as well

Reply:
I don't think any of this should be triggered for host toolchain, but better to be explicit here I think